README.md

```diff
@@ -51,17 +51,42 @@ Enable installation of packages from MELPA by adding an entry to
 Then just use `M-x package-list-packages` to browse and install
 packages from MELPA and elsewhere.
 
-Note that MELPA packages will always have higher versions than those
-from other archives like Marmalade, so if you decide you need
-non-MELPA versions of specific packages for some reason, extra
-configuration will be required:
+**Note:** Packages from the default “bleeding-edge” repository will
+always have higher versions than those from other archives like
+Marmalade, so if you decide you need non-MELPA versions of specific
+packages for some reason, extra configuration will be required:
 
-If your Emacs has the variable `package-pinned-packages`, you can
-customize or modify that variable as needed. Otherwise, use the
-separate
-[package-filter.el](https://github.com/milkypostman/package-filter)
-package which we provide.
+* If your Emacs has the variable `package-pinned-packages` (available
+  in 24.4 and later), you can customize or modify that variable as
+  needed.
 
+* You can use the
```

Comment:
There's a third option (Stable) so that didn't seem appropriate, to me at least.